assets/src/blocks/Accordion/AccordionEditor.js

```diff
@@ -0,0 +1,314 @@
+
+    return <Fragment>
+        <section>
+          <div className="block accordion-block my-3">
+            <header>
+              <RichText
+                tagName="h2"
+                className="page-section-header mt-3"
+                placeholder={__('Enter title for the block or leave empty to hide it.', 'planet4-blocks-backend')}
+                value={attributes.accordion_title}
+                onChange={this.toAttribute('accordion_title')}
+                keepPlaceholderOnFocus={true}
+                withoutInteractiveFormatting
+                characterLimit={100}
+                multiline="false"
+              />
+              <RichText
+                tagName="p"
+                className="page-section-description"
+                placeholder={__('Enter the description of the block or leave empty to hide it on the front page.', 'planet4-blocks-backend')}
+                value={attributes.accordion_description}
+                onChange={this.toAttribute('accordion_description')}
+                keepPlaceholderOnFocus={true}
+                withoutInteractiveFormatting
+                characterLimit={400}
+                multiline="false"
+              />
+            </header>
+
+            {[accordion_rows].map((accordion_row, index) => {
+              return (
+                <div key={index}>
+                  <div className="accordion-content my-2">
+                    <div className="accordion card-header"
+                        onClick={this.handleCollapseClick}
+                        id={attributes.accordion_id, index+1}
+                        >
+                      <RichText
+                        label={__('Row %s: Header', 'planet4-blocks-backend')}
+                        tagName="h4"
+                        className="accordion-headline"
+                        placeholder={__('Write title for the block here', 'planet4-blocks-backend')}
+                        value={attributes.accordion_headline}
+                        onChange={this.toAttribute('accordion_headline')}
+                        keepPlaceholderOnFocus={true}
+                        withoutInteractiveFormatting
+                        // characterLimit={100}
+                        multiline="false"
+                      />
+                    </div>
+                    <div className="panel">
+                      <RichText
+                        // id={attributes.accordion_id, index+1}
+                        tagName="p"
+                        className="accordion-text"
+                        placeholder={__('Enter description here', 'planet4-blocks-backend')}
+                        value={attributes.accordion_text}
+                        onChange={this.toAttribute('accordion_text')}
+                        keepPlaceholderOnFocus={true}
+                      />
+                    <Tooltip text={__('Add button text or leave empty to hide it on the front page.', 'planet4-blocks-backend')}>
```

Comment:
Again, here if we want to be consistent with the [Articles block](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/master/assets/src/blocks/Articles/ArticlesEditor.js#L134) we probably need to change the copy to `Edit text` for the tooltip and `Enter text` for the input placeholder... Actually we probably don't even need the tooltip here, we added it to Articles block only because there was a default value but here that's not the case!

Reply:
Sure I fixed the placeholder text;
The tooltip I added for the button so that it's clear that if there is no text the button won't show up on the front end. I understood that this is optional, also in our use case for the Nordics ( that I added accordion block as a custom html block atm) - we use this block for FAQ and we don't use button anywhere to be honest.. I think this "optionality" for the button needs to be discussed further .. :)